packages/workbox-background-sync/test/browser/background-sync-queue.js

```diff
@@ -57,60 +57,65 @@ describe('background sync queue test', () => {
     await backgroundSyncQueue.cleanupQueue();
   });
 
-  it('check defaults', () => {
+  it('should get default values if nothing is given in constructor', () => {
```

Comment:
Fine with me to do it gradually. I was mainly just showing how I like to structure my unit tests to more easily ensure the full public API space is covered.